index.html

```diff
@@ -2,15 +2,86 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Cake webpage</title>
+  <!-- Add a link to your css file here -->
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/9569a39f56.js" crossorigin="anonymous"></script>
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+  <!-- Add your markup here -->
+  <header id="header">
+<section class="header">
+    <img src="./img/cakelogo.png" alt="K-logo" class="logo" width="70px" />
+    <button class="hamburger" id="hamburger">
+      <i class="fas fa-bars"></i>
+    </button>
+    <p class="intro">The best cakes in town delivered to your door</P>
```

Comment:
Watch your capitalization on the closing tag </p>

Reply:
Corrected